docs/source/session.rst

```diff
@@ -0,0 +1,138 @@
+Sparkly Session
+===============
+
+``SparklySession`` is the main entry point to sparkly's functionality.
+It's derived from ``SparkSession`` to provide additional features on top of the default session.
+The are two main differences between ``SparkSession`` and ``SparklySession``:
+
+    1. ``SparklySession`` doesn't have ``builder`` attribute,
+       because we prefer declarative style over imperative.
+    2. Hive support is enabled by default.
+
+The next snippet shows how declarative style is used:
```

Comment:
[optional] The example shows both imperative and declarative. Instead of `style` I'd prefer word `definition`